packages/clutter_gtk.rb

```diff
@@ -0,0 +1,29 @@
+require 'package'
+
+class Clutter_gtk < Package
+  description 'The Clutter Gtk package is a library providing facilities to integrate Clutter into GTK+ applications.'
+  homepage 'https://wiki.gnome.org/Projects/Clutter'
+  compatibility 'all'
+  @_app = 'clutter-gtk'
+  @_ver = '1.8'
+  @_patch = '.4'
+  @_url = "https://download.gnome.org/sources/#{@_app}/#{@_ver}/#{@_app}-#{@_ver}#{@_patch}"
+  version @_ver + @_patch
+  source_url "#{@_url}.tar.xz"
+  source_sha256 `curl -Ls #{@_url}.sha256sum | tail -n1 | cut -d ' ' -f1`.chomp
```

Comment:
These need to be actual hashes, not the code which would generate a sha256 hash post-hoc.

Reply:
I think the checksum is a normal file and won't be changed:
```
gtk+-3.24.20.sha256sum | 170 B | 2020-Apr-27 15:12
gtk+-3.24.21.sha256sum | 170 B | 2020-Jun-28 21:36
```

Reply:
This is actually a good idea from @supechicken.  We would only need to bump the version on every update.  The problem is not all sources come with a sha256 hash so we can't use it for every package.

Reply:
My apologies to @supechicken ... I've been too dense to realize you're downloading the sha256sums for these gnome files from the archive directory.

Reply:
Carry on...

Reply:
We could do this for sources that don't have a sha256 checksum file:
```
source_sha256  `curl -Ls #{source_url} | sha256sum | cut -d' ' -f1`.chomp
```

Reply:
> We could do this for sources that don't have a sha256 checksum file:
> 
> ```
> source_sha256  `curl -Ls #{source_url} | sha256sum | cut -d' ' -f1`.chomp
> ```

Wait... The purpose of the checksum checking is to verify the file we downloaded, This command will generate a checksum according to the downloaded file, right?

Reply:
Yes, but we would have to do that anyway to determine the sha256 checksum if there wasn't a `.sha256` file available, right?